lib/charms/operator_libs_linux/v0/apt.py

```diff
@@ -124,7 +124,7 @@
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 3
+LIBPATCH = 4
```

Comment:
I'm assuming this is coming in from running `charmcraft fetch-lib`?

Reply:
yes!